_posts/2018-08-22-android-update.md

```diff
@@ -0,0 +1,43 @@
+---
+layout:     post
+title:      Servo for Android: nightly builds available and contribution opportunities
+date:       2018-08-22 09:00:00
+summary:    Servo for Android: nightly builds available and contribution opportunities
+categories:
+---
+
+We [recently revamped](https://github.com/servo/servo/pull/20912) Servo on Android.
+
```

Comment:
Is there a preview you could put here that shows something compelling? Either a quick video/gif of the product working or of a side-by-side of page scroll against a competing browser on the same platform? Something that will help get people excited, above and beyond the APK availability (which is totally awesome, btw!)

Reply:
There are screenshots. As for getting people excited, we are not delivering much in term of user experience. There's not much to show.